static/app/views/usageStats/usageStatsOrg.tsx

```diff
@@ -303,47 +310,41 @@ class UsageStatsOrganization extends AsyncComponent<Props, State> {
       },
       {
         title: t('Accepted'),
-        description: tct(
-          'Accepted [dataCategory] were successfully processed by Sentry.',
-          {dataCategory}
-        ),
+        help: tct('Accepted [dataCategory] were successfully processed by Sentry.', {
```

Comment:
nit: this tooltip has a period at the end while others dont 🙃

Reply:
```suggestion
        help: tct('Accepted [dataCategory] were successfully processed by Sentry', {
```